scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl

```diff
@@ -232,35 +241,44 @@ task ScoreVariantAnnotations {
 		File extracted_training_vcf_index
 		File? interval_list
 		Array[File] model_files
+		Boolean use_allele_specific_annotations
 
 		Int memory_mb = 16000
 		Int command_mem = memory_mb - 1000
 	}
 	Int disk_size = ceil(size(vcf, "GB") *2 + 50)
 
 	command {
+		zgrep -v '#' ~{vcf} > empty.txt
 		set -e
 
-		ln -s ~{sep=" . && ln -s " model_files} .
+		if [ -s empty.txt ]; then
+			ln -s ~{sep=" . && ln -s " model_files} .
 
-		export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
+			export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
 
-		gatk --java-options "-Xmx~{command_mem}m" \
-			ScoreVariantAnnotations \
-			~{"-L " + interval_list} \
-			-V ~{vcf} \
-			-O ~{basename}.~{mode} \
-			--model-backend ~{model_backend} \
-			~{"--python-script " + python_script} \
-			--model-prefix ~{basename} \
-			~{annotations} \
-			--mode ~{mode} \
-			--resource:extracted,extracted=true ~{extracted_training_vcf} \
-			~{resource_args}
+			gatk --java-options "-Xmx~{command_mem}m" \
+				ScoreVariantAnnotations \
+				~{"-L " + interval_list} \
+				-V ~{vcf} \
+				-O ~{basename}.~{mode} \
+				--model-backend ~{model_backend} \
+				~{"--python-script " + python_script} \
+				--model-prefix ~{basename} \
+				~{annotations} \
+				~{if use_allele_specific_annotations then "--use-allele-specific-annotations" else ""} \
+				-mode ~{mode} \
+				--resource:extracted,extracted=true ~{extracted_training_vcf} \
+				~{resource_args}
+		else
+			echo "Input VCF was empty so we'll return the same VCF that was input"
+			ln -s ~{vcf} ~{basename}.~{mode}.vcf.gz
+			ln -s ~{vcf_index} ~{basename}.~{mode}.vcf.gz.tbi
+		fi
 	}
 	output {
-		File scores = "~{basename}.~{mode}.scores.hdf5"
-		File annots = "~{basename}.~{mode}.annot.hdf5"
+		File? scores = "~{basename}.~{mode}.scores.hdf5"
```

Comment:
Maybe a comment here or above that these files are not produced for empty input VCFs?